nextflow_schema.json

```diff
@@ -18,19 +18,11 @@
                     "help_text": "A design file with information about the samples in your experiment. Use this parameter to specify the location of the input files. It has to be a comma-separated file with a header row. See [usage docs](https://nf-co.re/sarek/usage#input).\n\nIf no input file is specified, sarek will attempt to locate one in the `{outdir}` directory. If no input should be supplied, i.e. when --step is supplied or --build_from_index, then set --input false",
                     "fa_icon": "fas fa-file-csv",
                     "schema": "assets/schema_input.json",
-                    "anyOf": [
-                        {
-                            "type": "string",
-                            "format": "file-path",
-                            "exists": true,
-                            "mimetype": "text/csv",
-                            "pattern": "^\\S+\\.csv$"
-                        },
-                        {
-                            "type": "boolean",
-                            "enum": ["false"]
-                        }
```

Comment:
wasn't this done so `--step` would work without `input`?

Reply:
just tried it locally with:
```
nextflow run . -profile test,docker --outdir results
```

followed by:
```
nextflow run . -profile test,docker --outdir results --input false --step recalibrate
```

and it worked without failures.

Reply:
Originally yes, but we fixed it in nf-validation, now parameters evaluating to false are not validated